src/engine.h

```diff
@@ -1,4 +1,25 @@
 // Define this class here.
+#include <vector>
+#include <string>
 class RandomEngine {
+    
+    private: 
+        uint8_t length = 0;
+        bool symbl = false;
+
+    public:
+        std::vector<uint8_t>indx = {};
+        std::string password = "";
+    // Constructors:
+        explicit RandomEngine(uint8_t l); // To set the generator for the specific length
```

Comment:
Divas has declared them in his PR. I suggest wait for that to be merged then remove these from your PR and only push definitions

Reply:
Ok, sir